Regression: CMake fails with mixed Clang frontends in v0.8.1

This issue is observed in v0.8.1 when running the CMake test in ForCAD. It appears to be a regression due to a change between the two versions.

```
-- The C compiler identification is Clang 20.1.8 with MSVC-like command-line
-- The CXX compiler identification is Clang 20.1.8 with GNU-like command-line
CMake Error at .../Platform/Windows-Clang.cmake:177 (message):
  The current configuration uses the Clang compiler tool with mixed frontend
  variants, both the GNU and in MSVC CL like variants. This is not
  supported. Use either clang/clang++ or clang-cl as all C, C++, CUDA,
  and/or HIP compilers.
```

--- a/README.md
+++ b/README.md
@@ -30,7 +30,7 @@ The selected Fortran compiler is installed along with the corresponding C and C+
 | gfortran         | gcc        | g++          |
 | ifx              | icx        | icx          |
 | lfortran         | clang-cl   | clang-cl     |
-| flang, flang-new | clang-cl   | clang++      |
+| flang, flang-new | clang-cl   | clang-cl     |
 
 **The following environment variables are automatically set:**
 

--- a/platform/win/flang-new.js
+++ b/platform/win/flang-new.js
@@ -125,7 +125,6 @@ export async function setup(version = '') {
   const packages = [
     version ? `flang=${version}` : 'flang',
     version ? `llvm=${version}` : 'llvm',
-    version ? `clangxx=${version}` : 'clangxx',
     version ? `clang-tools=${version}` : 'clang-tools',
     version ? `llvm-openmp=${version}` : 'llvm-openmp',
     version ? `lld=${version}` : 'lld'
@@ -180,22 +179,19 @@ export async function setup(version = '') {
   await _exec('flang', ['--version']);
   await _exec('where', ['clang-cl']);
   await _exec('clang-cl', ['--version']);
-  await _exec('where', ['clang++']);
-  await _exec('clang++', ['--version']);
-  endGroup();
 
   // Export compiler-related environment variables
   startGroup('Exporting compiler environment variables');
   const envVars = {
     FC: 'flang',
     CC: 'clang-cl',
-    CXX: 'clang++',
+    CXX: 'clang-cl',
     FPM_FC: 'flang',
     FPM_CC: 'clang-cl',
-    FPM_CXX: 'clang++',
+    FPM_CXX: 'clang-cl',
     CMAKE_Fortran_COMPILER: 'flang',
     CMAKE_C_COMPILER: 'clang-cl',
-    CMAKE_CXX_COMPILER: 'clang++',
+    CMAKE_CXX_COMPILER: 'clang-cl',
     INCLUDE: [join(prefix, 'Library', 'include'), process.env.INCLUDE || ''].filter(Boolean).join(';'),
   };
 

--- a/platform/win/flang.js
+++ b/platform/win/flang.js
@@ -125,7 +125,6 @@ export async function setup(version = '') {
   const packages = [
     version ? `flang=${version}` : 'flang',
     version ? `llvm=${version}` : 'llvm',
-    version ? `clangxx=${version}` : 'clangxx',
     version ? `clang-tools=${version}` : 'clang-tools',
     version ? `llvm-openmp=${version}` : 'llvm-openmp',
     version ? `lld=${version}` : 'lld'
@@ -180,22 +179,19 @@ export async function setup(version = '') {
   await _exec('flang', ['--version']);
   await _exec('where', ['clang-cl']);
   await _exec('clang-cl', ['--version']);
-  await _exec('where', ['clang++']);
-  await _exec('clang++', ['--version']);
-  endGroup();
 
   // Export compiler-related environment variables
   startGroup('Exporting compiler environment variables');
   const envVars = {
     FC: 'flang',
     CC: 'clang-cl',
-    CXX: 'clang++',
+    CXX: 'clang-cl',
     FPM_FC: 'flang',
     FPM_CC: 'clang-cl',
-    FPM_CXX: 'clang++',
+    FPM_CXX: 'clang-cl',
     CMAKE_Fortran_COMPILER: 'flang',
     CMAKE_C_COMPILER: 'clang-cl',
-    CMAKE_CXX_COMPILER: 'clang++',
+    CMAKE_CXX_COMPILER: 'clang-cl',
     INCLUDE: [join(prefix, 'Library', 'include'), process.env.INCLUDE || ''].filter(Boolean).join(';'),
   };
 

